packages/client/screens/MusicPlaylistEditorCreationForm/__tests__/CreateMpeRoom.test.tsx

```diff
@@ -724,6 +735,12 @@ const createMpeRoomWithSettingsTestModel = createTestModel<
     },
 });
 
+//Could this be improve ?
+const playlistTotalDuration = datatype.number({
+    min: 42000,
+    max: 4200000,
+});
```

Comment:
When you execute a path generated by @xstate/test, by calling `path.test`, you can provide a `TestingContext`.
The playlist duration could be part of the `TestingContext`.
Therefore the duration would be computed in each test and would be different for each test.

Reply:
Nice!